### PR TITLE
Update the link of usage section in `cloudhub.md`

### LINKED ISSUE #2042: Error link of cloudhub in docs
**What would you like to be added/modified**:
the 3rd link of [Usage section of cloudhub](https://github.com/kubeedge/kubeedge/blob/master/docs/components/cloud/cloudhub.md#usage) is wrong, should be [start-the-websocket-and-quic-server-at-the-same-time](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-websocket-and-quic-server-at-the-same-time) instead of [start-the-quic-server-only](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-quic-server-only).
**Why is this needed**:
It is misleading.

### PR DESCRIPTION
**Pull request by llhuii**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Fix the wrong link of docs.
**Which issue(s) this PR fixes**:

Fixes #2042

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: docs/components/cloud/cloudhub.md [hunk 1]
```diff
@@ -62,4 +62,4 @@ The CloudHub can be configured in three ways as mentioned below :
 
 - **Start the websocket server only**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-websocket-server-only) to see the details.
 - **Start the quic server only**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-quic-server-only) to see the details.
-- **Start the websocket and quic server at the same time**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-quic-server-only) to see the details
\ No newline at end of file
+- **Start the websocket and quic server at the same time**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-websocket-and-quic-server-at-the-same-time) to see the details
```
Comment (daixiang0) [inline docs/components/cloud/cloudhub.md:65]: use relative path here.

**Comment by fisherxu** [comment]
@llhuii In the commit message, better not point to any issue :) The bot will say it `invalid commit messages` :)

File: docs/components/cloud/cloudhub.md (hunk 1 above)
Comment (llhuii) [reply docs/components/cloud/cloudhub.md:65]: ok

Comment (llhuii) [reply docs/components/cloud/cloudhub.md:65]: BTW there are other places having this issue, e.g. [README.md](https://github.com/kubeedge/kubeedge#in-the-cloud)

**Comment by llhuii** [comment]
> @llhuii In the commit message, better not point to any issue :) The bot will say it `invalid commit messages` :)

Got it.

File: docs/components/cloud/cloudhub.md (hunk 1 above)
Comment (fisherxu) [reply docs/components/cloud/cloudhub.md:65]: We can fixe them in a series of follow-up prs :)

**Comment by fisherxu** [comment]
/lgtm
/approve
Thanks for the fixing!

**Comment by fisherxu** [comment]
> > @llhuii In the commit message, better not point to any issue :) The bot will say it `invalid commit messages` :)
> 
> Got it. What's the logic of `invalid commit messages`? Any reference?

@llhuii  We use the prow-bot from k8s, the `invalid commit messages` is also from there :)
Ref: https://github.com/kubernetes/test-infra/pull/9387

**Pull request merged**